bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -41,9 +45,16 @@ def __init__(self,
         """
 
         self.runfolder_input = runfolder_input
-        self.samplesheet_file = runfolder_input + "/SampleSheet.csv"
         self.base_calls_input = runfolder_input + "/Data/Intensities/BaseCalls"
 
+        if not samplesheet:
```

Comment:
For now I think that this should be alright... I agree that it would be better if we had something that we could actually validate against, but I think that if we are to do something like that it needs to go into a separate module and do some fancy stuff so I think that it's out of scope for the PR. Go ahead and merge when you feel like it's ok. :)